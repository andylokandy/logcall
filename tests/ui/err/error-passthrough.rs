#[logcall::logcall]
fn f() {}

#[logcall::logcall]
fn f() {}

fn main() {}
