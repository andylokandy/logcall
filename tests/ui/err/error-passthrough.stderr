error[E0428]: the name `f` is defined multiple times
 --> tests/ui/err/error-passthrough.rs:5:1
  |
2 | fn f() {}
  | -- previous definition of the value `f` here
...
5 | fn f() {}
  | ^^ `f` redefined here
  |
  = note: `f` must be defined only once in the value namespace of this module
